docs/docs/actinia_concepts.md

```diff
@@ -0,0 +1,209 @@
+Actinia Basics
+==============
+
+Actinia REST API documentation
+------------------------------
+
+Actinia is fully documented using the OpenAPI standard[^1], better known
+as swagger[^2]. The JSON definition of the API can be accessed here:
+
+ <https://actinia.mundialis.de/latest/swagger.json>
+
+The full API documentation is available here:
+
+ <https://actinia.mundialis.de/api_docs/>
+
+To generate a readable documentation out of the swagger.json file, the
+spectacle tool can be used:
+
+```bash
+ # Download the latest swagger definition from the actinia service
+ wget  https://actinia.mundialis.de/api/latest/swagger.json -O /tmp/actinia.json
+
+ # Run spectacle docker image to generate the HTML documentation
+ docker run -v /tmp:/tmp -t sourcey/spectacle spectacle /tmp/actinia.json -t /tmp
+
+ # Start Firefox to show the documentation
+ firefox /tmp/index.html
+```
+
+The petstore swagger UI creator[^3] can be used to show all available
+REST API calls and all response models in a convenient way.
+
+**Footnotes**
+
+User, user-roles and user-groups
+--------------------------------
+
+Actinia use the concept of users, user-roles and user-groups to manage
+the access to the actinia databases and API calls. A single user has
+always a specific user role and can be member of a single user-group.
+
+The following user-roles are supported:
+
+ 1.  
+
+     superadmin
+
+     :   -   Can create, modify and delete users with all user-roles
+         -   Has access to all API calls and read/write access to all
+             databases
+
+ 2.  
+
+     admin
+
+     :   -   Can create, modify and delete locations in a user specific
+             database
+         -   Can access all API calls
+         -   Can create, modify and delete users with the maximum
+             user-role *user* of the same user group
+         -   Has access to persistent databases that were granted by a
+             superadmin
+
+ 3.  
+
+     user
+
+     :   -   Can run computational tasks in ephemeral and user specific
+             databases
+         -   Can create create, modify and delete mapsets in user
+             specific databases
+         -   Has limited acces to API calls
+         -   Can not create, modify or delete users
+         -   Has access to persistent databases that were granted by a
+             superadmin
+
+ 4.  
+
+     guest
+
+     :   -   Has very limited access to API calls
+         -   Can not create, modify or delete mapsets
+         -   Can not create, modify or delete users
+         -   Has access to persistent databases that were granted by a
+             superadmin
+
+Overview table:
```

Comment:
Some words in the overview table are wrapped at strange points, e.g.
p
ersistent

Can this be fixed in some automatic way?

Reply:
the table has been fixed and formatted